src/examples/forms/checkbox/Default.tsx

```diff
@@ -0,0 +1,29 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React from 'react';
+import { Field, Label, Checkbox } from '@zendeskgarden/react-forms';
+import { Row, Col } from '@zendeskgarden/react-grid';
+
+const Example = () => (
+  <Row justifyContent="center">
+    <Col size="auto">
+      <Field>
+        <Checkbox defaultChecked>
+          <Label>Annual</Label>
+        </Checkbox>
```

Comment:
Should we show these as controlled inputs for the default example? Otherwise it might not be obvious that they map to a native `<input />`

Reply:
Hmm, I didn't think it was necessary to show the default example as a controlled component. 

How does a controlled example demonstrate that a native input backs the component? Is it because native attributes are passed, such as `checked`? `defaultChecked` is also a native input attribute.

If it's important that the audience knows that `<Checkbox>` is backed by a native input element, I think the doc should explicitly communicate that in the component description section (which is currently blank).

Reply:
The majority of forms usage is "controlled" at this point. I think showing both a controlled and uncontrolled usage could be useful.

> How does a controlled example demonstrate that a native input backs the component

Specifically the `onChange` event shows the properties required to handle this. The `material-ui` [Checkbox example](https://material-ui.com/components/checkboxes/#checkbox) has a decent example of this. Probably not necessary for every example though.

Reply:
Current Garden also has an example. Won't speak to the quality :wink: https://garden.zendesk.com/react-components/forms/#advanced